_includes/articles/payment-batching.md

```diff
@@ -69,8 +66,8 @@ scenario.
 
 In addition to payment batching directly providing a fee savings,
 batching also uses limited block space more efficiently by reducing the
```

Comment:
perhaps /limited/the limited/

Reply:
I've changed this to "the limited". @harding - let me know if you disagree and think it should be reverted.